plugins/outputs/elasticsearch/README.md

```diff
@@ -199,6 +199,13 @@ This plugin will format the events in the following way:
   ## If set to true a unique ID hash will be sent as sha256(concat(timestamp,measurement,series-hash)) string
   ## it will enable data resend and update metric points avoiding duplicated metrics with diferent id's
   force_document_id = false
+
+  ## Specifies the handling of NaN and Inf values.
+  ## This option can have the following values:
+  ##    none -- do not modify field-values (default); will produce an error if NaNs or infs are encountered
+  ##    drop -- drop fields containing NaNs or infs
+  ##    <any float> -- Replace NaNs and inf with the given number and -inf with the negative number
+  # nan_handling = "none"
```

Comment:
The name `nan_handling` makes it sound like it only applies to NaN.

Since this applies to infinite values too, would it make sense to name this something more general, like `float_special_value_handling` or `float_handling`?

Reply:
You are absolutely right, changed to `float_handling`...